// guide/dist/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _fromGuide = require('./fromGuide');

Object.defineProperty(exports, 'fromGuide', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_fromGuide).default;
  }
});

var _toGuide = require('./toGuide');

Object.defineProperty(exports, 'toGuide', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_toGuide).default;
  }
});

var _errors = require('./errors');

Object.defineProperty(exports, 'GuideError', {
  enumerable: true,
  get: function get() {
    return _errors.GuideError;
  }
});
Object.defineProperty(exports, 'InvalidGuideError', {
  enumerable: true,
  get: function get() {
    return _errors.InvalidGuideError;
  }
});
Object.defineProperty(exports, 'UnexpectedNamespaceValueError', {
  enumerable: true,
  get: function get() {
    return _errors.UnexpectedNamespaceValueError;
  }
});
Object.defineProperty(exports, 'UnexpectedResourceTypeNameValueError', {
  enumerable: true,
  get: function get() {
    return _errors.UnexpectedResourceTypeNameValueError;
  }
});

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
